### PR TITLE
consolidate position of refresh bar

### PR DESCRIPTION
**Pull request by vincenzocaruso**
# Type of change? 

- [x] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 

...

**Checklist:**

- [ ] All tests passed
- [x] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/components/pages/dashboard/panels/analytics/analyticsPanel.scss [hunk 1]
```diff
@@ -3,6 +3,8 @@
 @import 'src/styles/mixins';
 @import 'src/styles/themes';
 
+$read-more: 20px;
```
Comment (vishwam) [inline src/components/pages/dashboard/panels/analytics/analyticsPanel.scss]: boo... :) use font size or gutter size variables

File: src/components/pages/dashboard/panels/analytics/analyticsPanel.scss [hunk 2]
```diff
@@ -28,7 +36,9 @@
         display: flex;
         align-items: flex-end;
 
-        .analytics-percentage-sign { @include rem-font-size(40px); }
+        .analytics-percentage-sign {
+          @include rem-font-size(40px);
```
Comment (vishwam) [inline src/components/pages/dashboard/panels/analytics/analyticsPanel.scss]: remove, use the normal font variables directly

File: src/components/shared/refreshBar/refreshBar.scss [hunk 3]
```diff
@@ -12,16 +14,24 @@ $rotateTime: 2s;
...
     @include rem-fallback(min-width, 0px);
 
-    &:hover { background: none !important; }
+    &:hover {
+      background: none !important;
+    }
 
-    .btn-icon { @include square-px-rem(15px); }
+    .btn-icon {
+      @include square-px-rem(15px);
```
Comment (vishwam) [inline src/components/shared/refreshBar/refreshBar.scss]: this is weird, why 15px?

File: src/components/shared/refreshBar/refreshBar.scss [hunk 4]
```diff
@@ -12,16 +14,24 @@ $rotateTime: 2s;
   font-size: 0.9em;
```
Comment (vishwam) [inline src/components/shared/refreshBar/refreshBar.scss]: remove, use font variable

File: src/components/shared/refreshBar/refreshBar.scss [hunk 5]
```diff
@@ -4,6 +4,8 @@
 @import 'src/styles/themes';
 
 $rotateTime: 2s;
+$minWidthTime: 238px;
```
Comment (vishwam) [inline src/components/shared/refreshBar/refreshBar.scss]: Why these numbers? They seem random

Comment (vincenzocaruso) [reply src/components/shared/refreshBar/refreshBar.scss]: it's the current design

File: src/components/shared/refreshBar/refreshBar.scss (hunk 3 above)
Comment (vincenzocaruso) [reply src/components/shared/refreshBar/refreshBar.scss]: updated to 16px

File: src/components/shared/refreshBar/refreshBar.scss [hunk 6]
```diff
@@ -4,24 +4,36 @@
...
+$minWidthEmptyTime:  33*$grid-size;
+$btn-icon-size: 4*$grid-size;
 
 .last-updated-container {
   display: flex;
   justify-content: flex-end;
   text-transform: uppercase;
-  font-size: 0.9em;
+  font-size: $font-size-default;
```
Comment (vishwam) [inline src/components/shared/refreshBar/refreshBar.scss]: remove? We shouldn't need this anymore

File: src/components/shared/timeSeriesInsightsLink/timeSeriesInsightsLink.js [hunk 7]
```diff
@@ -22,7 +22,7 @@ export class TimeSeriesInsightsLink extends Component {
     return (
       <div className="time-series-explorer-container">
         <Hyperlink href={href} onClick={this.onClick} target="_blank">{t('timeSeriesInsights.explore')}</Hyperlink>
-        <Tooltip position="bottom" content={
+        <Tooltip position={1} align={3} content={
```
Comment (vishwam) [inline src/components/shared/timeSeriesInsightsLink/timeSeriesInsightsLink.js]: Can't we use the enum value?

Comment (vincenzocaruso) [reply src/components/shared/timeSeriesInsightsLink/timeSeriesInsightsLink.js]: I'm doing a different PR to align this

**Review by vishwam — APPROVED** [review]

**Pull request merged**